adding ex 1

kardaszb opened the pull request.

kardaszb committed:
adding ex 1

pawel-javastart commented on src/wh_13_1/FileOperation.java:
```diff
@@ -0,0 +1,57 @@
+    protected FileOperation() {
+    }
+
+    static void WriteCSV(String fileName, String CSV_HEADER, List<Player> players) {
```
literówka : `WriteCSV` => `writeCSV`
CSV_HEADER tez powinien byc nazwan lowerCamelCasem, wtym kontekcie to nie jest stała tylko argument

kardaszb replied:
Poprawione

pawel-javastart commented on src/wh_13_1/ReadPlayers.java:
```diff
@@ -0,0 +1,41 @@
+    static List<Player> readFromUser() {
+        Scanner sc = new Scanner(System.in);
+        String readLine;
+        do {
+            System.out.println("Podaj wynik kolejnego gracza <Imie Nazwisko wynik> (lub stop):");
+            readLine = sc.nextLine();
+            if ("stop".equals(readLine) && players.size() == 0) {
+                break;
+            }
+            if ("stop".equals(readLine)) {
+                continue;
+            } else {
+                players = insertData(readLine);
+            }
+        } while (!"stop".equals(readLine));
```
troche duzo w tej petli sprawdzania czy "stop" zostalo podane. troche wporwadza to zamieszanie. Wydaje mi sie ze ladnie bedzie napisac tak:
```
while(true){
             readLine = sc.nextLine(); 
             if ("stop".equalsIgnoreCase(readLine)) { 
                    break;
             }
 ....
}
```

"stop" to tez dorby kandydat na stałą

kardaszb replied:
poprawione

kardaszb committed:
correcting:
WriteCSV => writeCSV
changing:
do - while loop to while(true) in read players class.
